doc/optimizers.md

```diff
@@ -1231,15 +1357,21 @@ Note that the `NesterovMomentumUpdate` class has the constructor
 
 #### Examples
 
+<details open>
+<summary>Click to collapse/expand example code.
+</summary>
+
 ```c++
 RosenbrockFunction f;
 arma::mat coordinates = f.GetInitialPoint();
 
 NesterovMomentumSGD optimizer(0.01, 32, 100000, 1e-5, true,
-    MomentumUpdate(0.5));
+  MomentumUpdate(0.5));
```

Comment:
```suggestion
    MomentumUpdate(0.5));
```

Oops, I think there was no need to change this one. :)

Reply:
should I change this ?